### PR TITLE
[GR] bump version

### PR DESCRIPTION
**Pull request by jheinen**
bump GR version to 0.72.10

### CODE REVIEW

**Comment by jheinen** [comment]
No idea how to fix this. Or should I stay with the old Qt 6.4.2 version?

File: G/GR/build_tarballs.jl [hunk 1]
```diff
@@ -99,7 +99,7 @@ dependencies = [
     Dependency("Libtiff_jll"; compat="~4.5.1"),
     Dependency("Pixman_jll"),
     HostBuildDependency("Qt6Base_jll"),
-    Dependency("Qt6Base_jll"; compat="~6.4.2"), # Never allow upgrading more than the minor version without recompilation
+    Dependency("Qt6Base_jll"; compat="~6.5.2"), # Never allow upgrading more than the minor version without recompilation
```
Comment (giordano) [inline G/GR/build_tarballs.jl:102]: @barche is it OK? Do you know what was the problem yesterday?

Comment (barche) [reply G/GR/build_tarballs.jl:102]: For sure it should be 6.5.2 so GR.jl and QML.jl can be used together, I'll take a look at the errors.

The problem yesterday was using GR_jll binaries compiled against Qt 6.4.2 together with Qt6Base_jll version 6.5.2, which normally should have been prevented by the compat entry, except that the compat entry was not strict enough.

**Comment by barche** [comment]
I set GCC to version 10, this should fix the error.

**Comment by barche** [comment]
Joy oh joy, the new Qt6.5 package is missing the XCB platform and can't work on Linux X11. I missed that because I did all this on my other machine, which is on Wayland.

**Comment by jheinen** [comment]
@barche : Do you think the draft status can be removed now with the successful build of Qt6 (including XCB extensions)?

**Comment by barche** [comment]
Testing now with the new Qt6Base binaries and the compiled artifact from this PR plotting with GR gives the following error. I have no immediate ideas on how to solve this:

```
.julia/dev/GR_jll/override/bin/gksqt: /usr/bin/../lib/julia/libcurl.so.4: version `CURL_OPENSSL_4' not found (required by /usr/lib/libproxy/libpxbackend-1.0.so)
connect: Connection refused
GKS: can't connect to GKS socket application
```

**Comment by giordano** [comment]
You're using a system library, that's the problem.

**Comment by barche** [comment]
Correct, I was accidentally using the Julia from arch linux. Works fine with the official binary for me now.

**Comment by giordano** [comment]
How did that happened though?

**Comment by barche** [comment]
Also the downloaded julia 1.9.3 official binary works fine with the artifacts from this PR, so whatever it is, it is something in my installation getting mixed up. I think it's safe to merge this PR.

**Comment by jheinen** [comment]
LGTM - I will bump `GR.jl` as soon as #7371 is merged.

**Comment by jheinen** [comment]
Thanks!

**Comment by charperflow** [comment]
@barche  Did you ever figure out how your system libraries were getting pulled in? I am currently running into the same issue and only have julia-bin installed from the AUR!

**Pull request merged**